configure.ac

```diff
@@ -987,6 +992,33 @@ if test "x$use_ccache" = "xyes"; then
     AX_CHECK_PREPROC_FLAG([-Qunused-arguments],[CPPFLAGS="-Qunused-arguments $CPPFLAGS"])
 fi
 
+dnl bignum value
+if test x"$req_bignum" = x"auto"; then
+  GMP_CHECK
+  if test x"$has_gmp" = x"yes"; then
+    set_bignum=gmp
+  fi
+
+  if test x"$set_bignum" = x; then
+    set_bignum=no
```

Comment:
Shouldn't this be
```autoconf
set_bignum=openssl
```
to satisfy the check further down? Otherwise the subsequent check will default to returning an error